testing/newt/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=newt
+pkgver=1.5.0
+_ver=1_5_0_tag
+pkgrel=0
+pkgdesc="Apache Newt is a smart build and package management tool for Apache Mynewt Operating System"
+url="https://mynewt.apache.org"
+arch="x86_64 x86"
+license="Apache-2.0"
+depends="git"
+makedepends="go"
+subpackages="$pkgname-doc"
+source="newt-$pkgver.tar.gz::https://github.com/apache/mynewt-newt/archive/mynewt_${_ver}.tar.gz"
+builddir="$srcdir/mynewt-newt-mynewt_${_ver}"
+
+build() {
+	cd "$builddir"
+	./build.sh
+}
+
+check() {
+	cd "$builddir"/newt
+	./newt version
```

Comment:
Just showing a version is not suitable for a `check` function, because it usually only tests a very small amount of code. We prefer to mark the package with `options="!check"` if there is no test suite available.

Reply:
Done! Thank you!